DIFI_101_Tutorial.md

```diff
@@ -24,7 +24,7 @@ There are multiple ways you can use DIFI in your application.  If you are using
 
 We will now create an example DIFI signal data packet in Python, without requiring any dependencies other than numpy and matplotlib, to demonstrate how it works.
 
-For reference, and because the DIFI specifications are behind a pay-wall, the signal data packet structure is depicted below, using 4 bytes per line:
```

Comment:
Is this false or do you just not want it pointed out?

Reply:
The last 2 official versions (1.1 and 1.2) are accessible for free, so it's not true anymore. Seeing that statement while doing the tutorial for the first time confused me because I had the 1.1 document opened while following the tutorial.

Reply:
Looks like it's accessible for free but through this form? https://dificonsortium.org/standards/  although it says " individual i.e., gmail address will not be accepted".  Did you find the standard docs elsewhere?

Reply:
i just checked and although they specify to not use gmail, it worked using mine. As soon as you submit the form the site redirects you to the docs, so I'm not sure if they even use your email, at most they just store it.

Reply:
Ah, yep, I tried entering garbage info and it immediately gave me the PDF.  Very annoying but better than a pay wall!